packages/python/plotly/plotly/express/_doc.py

```diff
@@ -147,6 +147,11 @@
         colref,
         "Values from this column or array_like are used to assign marks to facetted subplots in the horizontal direction.",
     ],
+    facet_col_wrap=[
+        "int",
+        "Wrap the column variable at this width, so that the column facets span multiple rows.",
```

Comment:
I would say "number of columns. Facets are wrapped at this number to span multiple rows". Maybe it's just me, but since I'm not familiar with this kind of representation I had a hard time understanding what this was before I actually tried it.

Reply:
Done!